libs/sdk-ui-pivot/src/impl/data/dataSource.ts

```diff
@@ -165,7 +165,7 @@ export class AgGridDatasource implements IDatasource {
                     .then((data) => {
                         const dataView = this.config.dataViewTransform(data);
                         const dv = DataViewFacade.for(dataView);
-                        this.gridApiProvider()?.setInfiniteRowCount(dataView.totalCount[0]);
+                        this.gridApiProvider()?.setRowCount(dataView.totalCount[0]);
```

Comment:
why this?

Reply:
I was getting 
![warnings in the console ](https://user-images.githubusercontent.com/15248640/174249717-84123a05-a7be-4623-be0b-a9a764003696.png)